README.md

```diff
@@ -119,6 +119,79 @@ Connection: keep-alive
 
 This error shows the request is missing a required property `name` from the HTTP request body.
 
+#### Server Validation
+
+You can simulate a request to a real server and verify whether it is valid according to the spec
+by providing a `__server` query param.
+
+Take this minimalistic spec (oas3) example:
+
+```json
+{
+  "openapi": "3.0.0",
```

Comment:
It's a guess here but I think people would rather see YAML examples. Less squiggles more point.